samples/drivers/gnss/src/main.c

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2023 Trackunit Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/gnss.h>
+
+static const struct device *gnss = DEVICE_DT_GET(DT_ALIAS(gnss));
+
+static void gnss_data_callback(const struct device *dev, const struct gnss_data *data)
+{
+	volatile int test;
+
+	test = 0;
```

Comment:
Shouldn't the sample do something in this callback, like log the navigation data to the console? I think it's confusing to have a sample that does nothing visible :)

Reply:
Nevermind, I just saw your last comment  :)
> The sample application will be extended to print the gnss data received, and dynamically adapt to the features supported by the GNSS modem.